Installation.md

```diff
@@ -282,27 +306,21 @@ ScriptAlias /git/ /usr/lib/git-core/git-http-backend/
 
 Your authorization script should first authenticate the user and then it can check whether the user is authorized to view a given git repository using either the `check_repo_permissions.rb` script or the `.access` file (deprecated):
 
-##### User authorization using the check_repo_permissions.rb script
+##### User authorization using the check_repo_permissions function
 
-When setting up authorization protocols for this access, your authorization script should check who has permission to access which git repos by executing the `bin/check_repo_permissions.rb` script.
+When setting up authorization protocols for this access, your authorization script should check who has permission to access which git repos by calling the `check_repo_permissions` postgresql function. This function takes 3 arguments:
 
-```sh
-./bin/check_repo_permissions.rb [user name] [relative path to git repo]
-```
+- the user name of the user requesting access to the instance
+- the course name
+- the repository name
 
-Where the `[user name]` is the user name of the person requesting access to the git repository and `[relative path to git repo]` is a relative path to the git repository. This relative path should only include the course name and the `.git` repository directory name. For example:
+For example:
 
 ```sh
-./bin/check_repo_permissions.rb student123 csc108/somerepo.git
+echo "SELECT check_repo_permissions(:'user_name', :'course_name', :'repo_name')" | psql -qtA -v user_name=student123 -v course_name=csc108 -v repo_name=somerepo
 ```
 
-This checks if the user with username `student123` has access to the somerepo.git repository in the course whose name is `csc108`.
-
-> :warning: **WARNING:** When running in a production environment you must also specify the `RAILS_ENV=production` environment variable when calling the `check_repo_permissions.rb` script. This is because the script makes a query to the database directly and so it needs to know which database to target. The above example in a production environment should therefore be changed to:
->
->  ```sh
->  RAILS_ENV=production ./bin/check_repo_permissions.rb student123 csc108/somerepo.git
->  ```
+> :warning: **WARNING:** If you receive an error that this function does not exist. Make sure that it is enabled by running the `db:functions` rake task in your production environment. This will build the function and make it available.
```

Comment:
I think this is from an older version of the setup, there's no `db:functions` rake task? (It's not clear to me whether this warning even makes sense, the MarkUs database should have the function just by loading our schema.)